src/components/doc/Sidebar.js

```diff
@@ -79,6 +79,7 @@ const DocSidebar = ({ location }) => {
             />
             <SidebarItem to="/docs/integrations/sentry/" text="Sentry" location={location} />
             <SidebarItem to="/docs/integrations/circleci/" text="CircleCI" location={location} />
+            <SidebarItem to="/docs/integrations/jira-token/" text="Jira" location={location} />
```

Comment:
Can we shorten the URL to `/docs/integrations/jira/` please? I think it makes sense to keep all Jira related instruction on one page.

Reply:
:+1:

Reply:
Shortened